plugins/openweather/public/openweather_api.rb

```diff
@@ -0,0 +1,71 @@
+module AresMUSH
+    module Openweather
+      def self.is_enabled?
+        !Global.plugin_manager.is_disabled?("Openweather")
```

Comment:
I don't think you really need the is_enabled checks.  If they installed the plugin as an extra, presumably they want it to be enabled.

Reply:
I think I may keep this in, because of room_desc_builder.rb. It follows the same call/format for weather command. Also if it give them issues, they can disable the plugin and not have to deal with changing this file. 

    def self.weather(room)
        return nil if !AresMUSH::Openweather.is_enabled?
        w = Openweather.weather_for_area(room.area_name)
        w ? "%R%R#{w}" : nil
      end
</pre>

Reply:
Sure thing!